packages/theme/src/cli/services/pull.ts

```diff
@@ -13,6 +13,7 @@ import {fetchChecksums} from '@shopify/cli-kit/node/themes/api'
 import {renderSuccess} from '@shopify/cli-kit/node/ui'
 import {glob} from '@shopify/cli-kit/node/fs'
 import {cwd} from '@shopify/cli-kit/node/path'
+import {insideGitDirectory, isClean} from '@shopify/cli-kit/node/git'
```

Comment:
~~Am I losing my mind or does `insideGitDirectory` not exist? I know we have `ensureInsideGitDirectory` but I can't see where we export insideGitDirectory.~~

It's added in this PR 🤦🏻‍♂️